build.gradle

```diff
@@ -46,6 +46,8 @@ dependencies {
     implementation "io.springfox:springfox-swagger2:${springfoxSwaggerVersion}"
     implementation "io.springfox:springfox-swagger-ui:${springfoxSwaggerVersion}"
     implementation "javax.inject:javax.inject:1"
+    implementation "org.apache.tika:tika-core:1.22"
+    implementation "org.apache.tika:tika-parsers:1.22"
```

Comment:
✏️ Do these versions need to match?

Reply:
I actually only need one of those dependencies. I'll remove the other.